src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -79,7 +73,8 @@ export function sortMolecularProfilesForDisplay(profiles:MolecularProfile[]) {
 
 export const CNA_STROKE_WIDTH = 1.8;
 export const PLOT_SIDELENGTH = 650;
-
+export const WATERFALLPLOT_BASE_SIDELENGTH = 500;
+export const WATERFALLPLOT_SIDELENGTH_SAMPLE_MULTIPLATION_FACTOR = 1.6;
```

Comment:
like I said before, waterfall plot doesn't seem the right name

Reply:
also multiplication is misspelled

Reply:
Spelling error corrected. Awaiting outcome of waterfall plot name.